tests/test_override_system_js_lib_symbol.js

```diff
@@ -7,7 +7,18 @@ mergeInto(LibraryManager.library, {
 	glTexImage2D__deps: ['orig_glTexImage2D'],
 	glTexImage2D: function(target, level, internalFormat, width, height, border, format, type, pixels) {
 		_glTexImage2D.createdType = type;
-		_orig_glTexImage2D(target, level, internalFormat, width, height, border, format, type, pixels);
+		// Check that the orignal fuction exists
+		assert(_orig_glTexImage2D);
+		// Also try invoking glTexImage2D to verify that it is actually the
+		// underlying functions from library_webgl.js
```

Comment:
```suggestion
		// underlying function from library_webgl.js
```

Reply:
Ah... I thought the other PR had automerge but it didn't, then I thought this PR didn't but it did... anyhow, this nit isn't worth a new PR, please ignore...